presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -210,6 +221,12 @@ Accessors
 
     Return the Y coordinate of the point.
 
+.. function:: ST_NumGeometries(Geometry) -> bigint
+
+   If geometry is a GEOMETRYCOLLECTION (or MULTI*) returns the number of geometries,
```

Comment:
To be consistent with the above function doc: `If the geometry is a collection of geometries (e.g., GEOMETRYCOLLECTION or MULTI*)`